Subdivision_method_3/test/Subdivision_method_3/test_Subdivision_method_3.cpp

```diff
@@ -1,361 +1,236 @@
-// ============================================================================
-//
-// Copyright (c) 2005-2006, 2017 Le-Jeng Shiue
-//
-// This software and related documentation is part of an INTERNAL release
-// of the Computational Geometry Algorithms Library (CGAL). It is not
-// intended for general use.
-//
-// ----------------------------------------------------------------------------
-//
-// release       : $CGAL_Revision: $
-// release_date  : $CGAL_Date: $
-//
-// file          : test/Subdivision_method_3/test_Subdivision_method_3.C
-// package       : Subdivision_method_3
-// chapter       : Subdivision Method
-//
-// revision      : $Id$
-// revision_date : $Date$
-//
-// author(s)     : Le-Jeng Shiue <Andy.Shiue@gmail.com>
-//
-// Test subdivision methods
-// ============================================================================
+#include <CGAL/subdivision_method_3.h>
 
 #include <CGAL/Simple_cartesian.h>
 #include <CGAL/Polyhedron_3.h>
 #include <CGAL/Surface_mesh.h>
-
-#include <CGAL/subdivision_method_3.h>
+#include <CGAL/Polygon_mesh_processing/measure.h>
+#include <CGAL/Polygon_mesh_processing/triangulate_faces.h>
 
 #include <iostream>
 #include <fstream>
 #include <cassert>
 
-using namespace std;
-using namespace CGAL;
-
-#define TEST_DEPTH (3)
-
-//#define TESTMESH_GENERAL   "data/??.off"
+namespace PMP = ::CGAL::Polygon_mesh_processing;
+namespace SM = ::CGAL::Subdivision_method_3;
+namespace params = SM::parameters;
```

Comment:
Don't we now have a `CGAL::parameters` namespace ?

Reply:
Yes, I left it like this in tests to test backward compatibility